sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,736 @@
+-- Tol Barad --
+
+-- Spawns are located half way down
+SET @CGUID := 376000; -- 214 creature entries (380000->)
+SET @OGUID := 68000; -- 111 gameobject entries
```

Comment:
Based on TrinityCore rev. f6637439f484 2016-06-18 16:23:55 +0200 (6.x branch), in TDB 6.04 plus updates up to and including `2016_06_17_01_world.sql`, these values are available:
- 214 consecutive free GUIDs above 250000 in the `creature` table: 250970 - 251183
- 111 consecutive free GUIDs above 200000 in the `gameobject` table: 234355 - 234465